### PR TITLE
Example homepage with ocular gatsby

### PR DESCRIPTION
**Pull request by jckr**
This PR prepares luma.gl to have an example homepage w/ gatsby. This includes some changes to the ocular config. 

For #

#### Change List
-

### CODE REVIEW

File: website/ocular-config.js [hunk 1]
```diff
@@ -79,13 +97,13 @@ module.exports = {
...
-  siteUrl: "https://luma.gl", // Domain of your website without pathPrefix.
-  pathPrefix: "/luma", // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
-  siteRss: "/rss.xml", // Path to the RSS file.
-  dateFromFormat: "YYYY-MM-DD", // Date format used in the frontmatter.
-  dateFormat: "DD/MM/YYYY", // Date format for display.
-  userName: "WebGL User", // Username to display in the author segment.
-  copyright: "Copyright © 2017 Uber. MIT Licensed", // Copyright string for the footer of the website and RSS feed.
-  themeColor: "#c62828", // Used for setting manifest and progress theme colors.
-  backgroundColor: "#e0e0e0" // Used for setting manifest background color.
+  siteUrl: 'https://luma.gl', // Domain of your website without pathPrefix.
```
Comment (ibgreen) [inline website/ocular-config.js]: Can we cut these now? I don't think we use them, they are from the original starter.

Comment (jckr) [reply website/ocular-config.js]: yes. we should put them somewhere else eventually. let's remove them from ocular-config-template

on a side note - 
idk what to do w/ conflicts here,
i can't rebase cleanly bc the new example glfx is looking for something in @loaders.gl it doesn't get

Comment (ibgreen) [reply website/ocular-config.js]: You can comment out the glfx example from the website for now, it is under active development.

Comment (jckr) [reply website/ocular-config.js]: ok. re the static files (which are the thumbnails file), I'd rather have them in /src, or really anywhere but /static. the reasoning is that they are going to be transformed into much smaller images. only those smaller images will be exposed in the final site. imo, what's in /static is not expected to change and is made available as is

**Pull request merged**